include/RAJA/policy/atomic_builtin.hpp

```diff
@@ -225,6 +225,19 @@ RAJA_INLINE T builtin_atomic_CAS_oper_sc(T volatile *acc,
 
 }  // namespace detail
 
+template <typename T>
+RAJA_DEVICE_HIP
+RAJA_INLINE void atomicStore(builtin_atomic, T volatile *acc, T value)
+{
+  *acc = value;
```

Comment:
This should use something like atomicExchange to ensure actual atomicity.

Reply:
Yeah, this should definitely not be a bare store.